tests/e2e/cucumber/features/integrations/share.oc10.feature

```diff
@@ -47,6 +47,9 @@ Feature: share
       | resource          | to               | create_version |
       | PARENT/simple.pdf | folder_to_shared | true           |
     #Then "Alice" should see that the resource "folder_to_shared/simple.pdf" has 1 version
+    And "Brian" downloads old version of the following resource
```

Comment:
Kind of missing a `Then` step here, or are we just trying to download and would run into a failure if it doesn't work?

Reply:
yes, all checks are inside steps "And" or "When".  I honestly don't know why we left the commented "Then". maybe we'll delete commented steps later